### PR TITLE
Big Batch of Potpourri Updates for Delta [2.1]

### PR DESCRIPTION
**Pull request by Okand37**
**Atmospherics:**
Atmospheric's solar panel has been fixed. Lack of accepting atmospheric's ID and unconnected power wire are corrected.
The Atmospherics engine now contains an emergency escape. There is also a new canister storage which contains x1 Plasma Canister and x1 Nitrogen Canister.
The Supermatter's cooling loop has been expanded and redone, it looks better and should work better.
The Incinerator's air injector has been properly added to the area, ensuring it is powered at roundstart.
Atmospheric's wardrobe now has a few more winter coats.
**Engineering:**
Added CE's stamp to the CE office, and some additional winter coats to the engineering wardrobe.
Removed the containment and chamber blockades and cleaned up the area, adding more support for engineering to setup a P.A. and corresponding engine mid-round with cargo's assistance.
**Chapel:**
Added five logs to the chaplain's wardrobe as a small gimmick supply (bonfires!)
**Security:**
Added the missing canvases to the permabrig's easel and added decals under the rest of the security lockers in the brig. Additionally, also added some spare winter coats to the wardrobe.
**Medical:**
Turns morgue maintenance into a proper part of maintenance, complete with radiation shielding and fixed the lower left chemistry exterior windoor requiring chemistry access to open.
**Service:** 
The bartender can now enter the theatre stage from their backroom directly with a new door.
**Locker Room:** 
Rearranged the Locker Room to add spaces for the lockers, and redesigned Cabin 1!

:cl: Okand37 (DeltaStation Updates)
add: Tweaked Atmospherics
fix: Fixed the Incinerator air injector
add: Tweaked Engineering
add: Added logs to the Chaplain's closet for gimmicks (bonfires)
add: Tweaked Security
fix: Fixed medical morgue maintenance not providing radiation shielding and exterior chemistry windoor
add: Bar now has a door from the backroom to the theatre stage
add: Tweaked Locker Room/Dormitories
/:cl:

### CODE REVIEW

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 1]
```diff
@@ -7188,8 +7203,9 @@
 /obj/machinery/atmospherics/pipe/heat_exchanging/simple{
 	dir = 4
 	},
+/obj/machinery/atmospherics/pipe/heat_exchanging/simple,
 /obj/structure/lattice/catwalk,
-/turf/open/space,
+/turf/open/space/basic,
```
Comment (ShizCalev) [inline _maps/map_files/Deltastation/DeltaStation2.dmm]: There a reason these are switched to /basic?

Comment (Okand37) [reply _maps/map_files/Deltastation/DeltaStation2.dmm]: Nope, didn't actually know I did that. Might've happened when I was trying to remove the d1 and d2 variables or due to the most recent update with the white ship.

**Comment by Nabski89** [comment]
Not being able to use that atmos solar has literally gotten me killed twice. YAY!

**Comment by SyruseEln** [comment]
Delta whiteship is next right?

**Comment by Okand37** [comment]
@SyruseEln I was going to make one but someone else beat me to it. #32414

**Pull request merged**